src/FSharpx.Collections/NonEmptyList.fs

```diff
@@ -5,29 +5,27 @@ open System.Collections
 open System.Collections.Generic
 open System.Runtime.CompilerServices
 open FSharpx
-
+ 
 type NonEmptyList<'T> = 
-    { Head: 'T
-      Tail: 'T list }
+    private { List: 'T list }
```

Comment:
Ah good point.
On May 23, 2016 18:12, "varon" <notifications@github.com> wrote:

> In src/FSharpx.Collections/NonEmptyList.fs
> <https://github.com/fsprojects/FSharpx.Collections/pull/62#discussion_r64245848>
> :
>
> >  type NonEmptyList<'T> =
> > -    { Head: 'T
> > -      Tail: 'T list }
> > +    private { List: 'T list }
>
> Can do, but this would require implementing (and testing) an awful lot of
> things we get for free with a record type, such as structural equality,
> comparison, etc.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/fsprojects/FSharpx.Collections/pull/62/files/53ed3aaf66cdf3e2a03306595d91431558eae4dd#r64245848>
>